Documentation/contributing/development/season_of_docs.rst

```diff
@@ -0,0 +1,114 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+
+.. _season_of_docs:
+
+Google Season of Docs 2022
+==========================
+
+Cilium is interested in applying to the 2022 Google Season of Docs to improve the documentation experience for our users and contributors.
+Below is our project ideas, if you have any questions, please reach out to gsod@cilium.io.
+
+Reorganize Cilium Features and Getting Started 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+About Our Organization
+~~~~~~~~~~~~~~~~~~~~~~
+
+Cilium (current version 1.12, first released in 2017) is an Apache 2.0-licensed project for providing, 
```

Comment:
nit: For what it's worth, 1.12 is the current development version, 1.11 is the latest GA release at this time. We plan to release 1.12 during the program, possibly also 1.13 but we have not yet planned out anything for that version.

Reply:
thanks updated it to 1.11